viewer/core/src/public/migration/Cognite3DViewer.ts

```diff
@@ -457,6 +595,54 @@ export class Cognite3DViewer {
     }
   }
 
+  /**
+   * Sets camera controls mode anything allowed in CameraControlsOptions type.
+   * @param controlsOptions JSON object with camera controls options.
+   */
+  setCameraControlsMode(controlsOptions: CameraControlsOptions) {
+    this._cameraControlsOptions.useOnClickTargetChange =
+      controlsOptions?.onClickTargetChange ?? this._cameraControlsOptions.useOnClickTargetChange;
```

Comment:
`controlsOptions` can never be undefined, so no reason to use `?` here

Reply:
Yup